### ISSUE TITLE
SoTC crashes (differently) /w SPU asmjit & /w SPU Precise

### ISSUE THREAD

**Issue by elad335** [RPCS3/rpcs3#3671]
With SPU precise decoder the game crashes with a black screen after showing a few logos.
While with SPU asmjit the game goes further but crashes in the first frame (it succeeds to renders) of the cutscene ( for some reason everytime it crashes on a diffrent frame, but it still the first frame to be shown on screen).
also , with SPU asmjit it gets to the loading circle but it takes him a few minutes to finish and then there is flickering all over the screen .

[www.youtube.com/watch - SPU_Precise](https://www.youtube.com/watch?v=68O2vi_iR1c)
[www.youtube.com/watch - SPU_Asmjit](https://www.youtube.com/watch?v=6aHAvaVZZhs&feature=youtu.be)

[SPU_Precise.log.gz](https://github.com/RPCS3/rpcs3/files/1424312/SPU_Precise.log.gz)
[SPU_asmjit.log.gz](https://github.com/RPCS3/rpcs3/files/1424317/SPU_LLVM.log.gz)
[NPEA00280]
RPCS3 v0.0.3-3-dbb2251 Alpha | HEAD
Intel(R) Core(TM) i7-6700K CPU @ 4.00GHz | 4 Threads | 7.96 GiB RAM | AVX | TSX
gtx 980 ti
Windows 10

**Comment by elad335** [comment]
update : enabling WCB helps with the flickering and make the game go further , but colors are a bit off -
![image](https://user-images.githubusercontent.com/18193363/32135832-186cf6e4-bc0e-11e7-93cc-8c84459e26c6.png)

**Comment by money123451** [comment]
full pixel mode now just crashes the games emulation when switched to it on both vulkan and opengl now (its a option in the games option menu if you are not aware) and vulkan has had visual issues always for this game so opengl is recommended if you want correct colors but without full pixel mode its visuals are a bit messed up see first snapshot.
![2017-10-28 14_49_35-fps_ 14 57 _ opengl _ 0 0 3-3-head _ shadow of the colossus _ npua80677](https://user-images.githubusercontent.com/12605368/32137506-598ec93c-bbef-11e7-9ba4-deee07458f4e.png)

full pixel mode on with [this build](https://ci.appveyor.com/project/rpcs3/rpcs3/build/0.0.3-6588/artifacts) 
![2017-10-28 15_10_38-fps_ 13 13 _ opengl _ 0 0 3-3-head _ shadow of the colossus _ npua80677](https://user-images.githubusercontent.com/12605368/32137677-7374f620-bbf2-11e7-96a8-4bdd8b09f4cb.png)

[known working build log](https://www.dropbox.com/s/n6xevmu3xbcypbq/SOTCworking-RPCS3.log.gz?dl=0)
[Full pixel mode crash log](https://www.dropbox.com/s/hefz0xo48hq8kfi/RPCS3.log.gz?dl=0)
[build that was crashing and latest master at time of writing](https://ci.appveyor.com/project/rpcs3/rpcs3/build/0.0.3-6618)

**Comment by elad335** [comment]
update: it crashing on latest master with
```F {rsx::thread} class std::runtime_error thrown: Unknown surface depth format 0x0
(in file Emu\RSX\gcm_enums.cpp:655)```
[RPCS3.log](https://github.com/RPCS3/rpcs3/files/1618149/RPCS3.log)

**Comment by AniLeo** [comment]
Any updates?

**Comment by money123451** [comment]
oh god i forgot this was a issue existed and it seems that all issues have been resolved with changing full pixel mode on or off and visually it mostly looks fine now as well with no special settings

**Comment by elad335** [comment]
closing as the original issue here is fixed, further crash is reported at #5584.
